python_ta/reporters/html_reporter.py

```diff
@@ -14,6 +15,7 @@
 
 TEMPLATES_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'templates')
 
+
```

Comment:
There are a few style changes which are correct but beyond the scope of this PR. Please revert them here and make them in a separate pull request.